virtual_rainforest/models/abiotic/energy_balance.py

```diff
@@ -0,0 +1,321 @@
+"""The `abiotic.energy_balance` module.
+
+This module calculates the energy balance for the Virtual Rainforest.
+The sequence of processes is based on Maclean et al, 2021: Microclimc: A mechanistic
+model of above, below and within-canopy microclimate. Ecological Modelling
+Volume 451, 109567. https://doi.org/10.1016/j.ecolmodel.2021.109567.
+
+# --- this is the sequence of processes in microclimc for one time step ---
```

Comment:
Is this taken from the package you are extracting the equations from? If it is I think it might be worth contextualising this bit more in the docstring

Reply:
I'll put a more detailed description of the processes in the docstring or a separate documentation